packages/wallet-api-manifest-validator/tests/validator.test.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable */
+import { validateManifest } from "../src/validator";
+
+import * as fs from "fs";
+import path from "path";
+
+const manifestDapp = JSON.parse(
+  fs.readFileSync(
+    path.join(__dirname, "manifests/dapp-manifest.test.json"),
+    "utf-8"
+  )
+);
+
+const manifestWalletApp = JSON.parse(
+  fs.readFileSync(
+    path.join(__dirname, "manifests/walletApp-manifest.test.json"),
+    "utf-8"
+  )
+);
+
+const manifestwebBrowser = JSON.parse(
+  fs.readFileSync(
+    path.join(__dirname, "manifests/webBrowser-manifest.test.json"),
+    "utf-8"
+  )
+);
+
+test("Dapp's Manifest base is OK", () => {
```

Comment:
You should also test invalid manifests to make sure the validation schema catches them and detect all errors (no need to be 100% exhaustive at first, but make sure to test the critical fields like the required ones)